torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -515,10 +515,10 @@ def _is_supported(self, module: nn.Module, insert: bool = False) -> bool:
         Returns True if the module is supported by observer, False otherwise
         """
         # check to see if module is of a supported type
-        is_supported_type = sum([isinstance(module, x) for x in self.DEFAULT_DYNAMIC_STATIC_CHECK_SUPPORTED]) > 0
+        is_supported_type = sum(isinstance(module, x) for x in self.DEFAULT_DYNAMIC_STATIC_CHECK_SUPPORTED) > 0
```

Comment:
Same as above `sum(predicate) > 0` == `any(predicate)`
```suggestion
        is_supported_type = any(isinstance(module, x) for x in self.DEFAULT_DYNAMIC_STATIC_CHECK_SUPPORTED)
```